observer/probers/tls/tls.go

```diff
@@ -92,102 +84,80 @@ func getOCSPStatus(cert, issuer *x509.Certificate) (int, error) {
 		return ocsp.Unknown, err
 	}
 
-	switch ocspRes.Status {
-	case ocsp.Revoked:
-		return ocsp.Revoked, nil
-	case ocsp.Good:
-		return ocsp.Good, nil
-	default:
-		return ocsp.Unknown, nil
-	}
+	return ocspRes.Status, nil
 }
 
-// Make a TLS connection with InsecureSkipVerify set to true to get the
-// expiration time and root information of insecure leaf certificates.
-func (p TLSProbe) getInsecureCertInfo() (string, string, time.Duration, error) {
-	conf := &tls.Config{InsecureSkipVerify: true}
-	conn, err := tls.Dial("tcp", p.url+":443", conf)
+// Export expiration timestamp and reason to Prometheus.
+func (p TLSProbe) exportMetrics(notAfter time.Time, reason reason) {
+	p.notAfter.WithLabelValues(p.url).Set(float64(notAfter.Unix()))
+	p.notAfter.WithLabelValues(p.url, reasonToString[reason]).Inc()
+}
+
+func (p TLSProbe) probeExpired(timeout time.Duration) (bool, time.Duration) {
+	start := time.Now()
+	conn, err := tls.DialWithDialer(&net.Dialer{Timeout: timeout}, "tcp", p.url+":443", &tls.Config{InsecureSkipVerify: true})
 	if err != nil {
-		return "", "", 0, err
+		p.exportMetrics(time.Time{}, internalError)
+		return false, time.Since(start)
 	}
+	defer conn.Close()
```

Comment:
nit: insert an empty line after this. We like to structure all of our code as
```
do some computation
if some error check {
  exit
}
<empty line>
do some more computation...
```

In this case it's totally appropriate to pair the `defer conn.Close()` with the code above it, but we should have a break before we dive into defining `peers` and doing that loop.

Take a look through the code and find other places you can insert a blank line to separate logical blocks that end with an error check.